changelog.md

```diff
@@ -1,4 +1,9 @@
-# 6.1.4 - xx.xx.2025
+# 6.x.x - xx.xx.2025
+**What's New**
+* [MainMenu][Breaking Change]: converted to a functional component, removed `children` support, use `items` prop only, removed outdated customer logo(`customerLogoUrl`, `customerLogoLink`, `customerLogoHref`, `customerLogoBgColor`) and `isTransparent` props.
```

Comment:
Let's also add the link to the doc example about how provide `items` instead of `children`